doc/services/portability/posix/option_groups/index.rst

```diff
@@ -509,7 +509,7 @@ _XOPEN_STREAMS
     getpmsg(),
     ioctl(),yes
     isastream(),
-    putmsg(),
+    putmsg(),yes
```

Comment:
This is not correct, putmsg was not fully implemented and we just provide an empty stub, can we clarify please, otherwise it is false advertisement, "yes" should be used only for fully implemented functions IMO, for empty stubs, lets use something else to make it clear.

Reply:
Yes - good catch.

@moonlight83340 - can you please match what was done for other functions where ENOSYS is reported?

Also, in a second commit, please ensure that other ENOSYS functions have similar entries in the docs.

Reply:
in posix directory, I saw we still put the yes in the documentation even when there are ENOSYS 😓 
Is there any other part of the documentation that should be modify ? after a quick search for ENOSYS functions and documentation files, I'm not seing others changes.

Reply:
Please copy the .rst source where it says 

"(will fail with ENOSYS†)"

https://docs.zephyrproject.org/latest/services/portability/posix/conformance/index.html#posix-system-interfaces

Reply:
should I keep the yes with the "(will fail with ENOSYS†)" in the compliance documentation part ?

Reply:
> This is not correct, putmsg was not fully implemented and we just provide an empty stub, can we clarify please, otherwise it is false advertisement, "yes" should be used only for fully implemented functions IMO, for empty stubs, lets use something else to make it clear.

Do you think modifications are okay ? 😅